NodePhantomJS/Dockerfile

```diff
@@ -4,18 +4,24 @@
 # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 FROM selenium/node-base:3.4.0-einsteinium
 LABEL authors=SeleniumHQ
+MAINTAINER Ken.l.ma
+# used as a fork from the selenium docker images
+# update phantomjs to 2.0.0
```

Comment:
If this is a fork, change the maintainer in your own fork. we would not be changing the mantainer from SeleniumHQ